src/components/BrainfuckEngine.vue

```diff
@@ -106,19 +106,43 @@
 
     <form @submit.prevent="input" style="margin: 12px 0px">
       <b-field>
-        <b-input v-model="inputText" :disabled="!isRunning"> </b-input>
+        <b-input v-model="inputText" :disabled="isRunning && inPause">
+        </b-input>
         <div class="control">
-          <button class="button is-dark" :disabled="!isRunning">Input</button>
+          <button class="button is-dark" :disabled="isRunning && inPause">
+            Input Text
+          </button>
         </div>
       </b-field>
     </form>
 
+    <form @submit.prevent="inputCharCode" style="margin: 12px 0px">
+      <b-field>
+        <b-input
+          v-model.number="inputCode"
+          :disabled="isRunning && inPause"
+          type="number"
+          min="0"
+          max="255"
+          step="1"
+        >
+        </b-input>
+        <div class="control">
+          <button class="button is-dark" :disabled="isRunning && inPause">
+            Input Value
+          </button>
+        </div>
+      </b-field>
+    </form>
+
+    <b-checkbox v-model="repeatHistory"> Repeat History </b-checkbox>
```

Comment:
- Add `:disabled="isRunning` to this `<b-checkbox>`, because changing this checkbox has no effect while running.
- Move this `<b-checkbox>` to the bottom of `Input History` for UI/UX.